Assets/Project/Scripts/AbilitiesManager.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityChan;
+using UnityEngine;
+
+public class AbilitiesManager : MonoBehaviour
+{
+
+    [SerializeField] private PointSystem pointSystem;
+    [SerializeField] private UnityChanControlScript unityChan;
+
+    [SerializeField] private float secondsToEnableJumping = 10;
+    [SerializeField] private float secondsToEnableSliding = 20;
+    [SerializeField] private float secondsToEnableAttack = 30;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
private void Update( ) - let's keep functions consistent with private. You can change default script in the editor files if you want

Reply:
oh nice tip, will change the  default script